tests/utils/dockercli/volumelifecycle.go

```diff
@@ -39,6 +41,16 @@ func AttachVolume(ip, volName, containerName string) ([]byte, error) {
 		" busybox tail -f /dev/null")
 }
 
+// AttachVolumeWithRestart - attach volume to container on given host
+// this util starts the container with restart=always flag so that container
+// automatically restarts if killed
+func AttachVolumeWithRestart(ip, volName, containerName string) ([]byte, error) {
+	log.Printf("Attaching volume [%s] on VM[%s]\n", volName, ip)
+	return ssh.InvokeCommand(ip, dockercli.RunContainer+" --restart=always -d -v "+volName+
+		":/vol1 --name "+containerName+
+		" busybox tail -f /dev/null")
```

Comment:
minor: `busybox tail -f /dev/null` should move at const file .. used at L41 as well